pkgs/development/python-modules/crossplane/default.nix

```
@@ -0,0 +1,19 @@
+{ lib, buildPythonPackage, fetchFromGitHub }:
+
+buildPythonPackage rec {
+  pname = "crossplane";
+  version = "0.5.7";
+  src = fetchFromGitHub {
+    owner = "nginxinc";
+    repo = "crossplane";
+    rev = "v${version}";
+    sha256 = "0lv3frfvnvz5wjxwh3mwy8nbypv4i62v4bvy5fv7vd6kmbxy1q9l";
+  };
+
+  meta = with lib; {
+    homepage = "https://github.com/nginxinc/crossplane";
+    description = "NGINX configuration file parser and builder";
+    licenses = licenses.apsl20;
```

The license is [Apache Software License](https://github.com/nginxinc/crossplane/blob/master/LICENSE) not the Apple one.

```suggestion
    licenses = licenses.asl20;
```

---

Fixed.